tutorial/create-test-virtual-device.md

```diff
@@ -0,0 +1,212 @@
+
+The application is created and bootstrapped. Now, to see the application details and its status, run:
+
+      amc application list
+
+Watch the output of this command till the application becomes *ready*.
+
+## Test the virtual device
+
+**Launch an instance**
+
+When the application is *ready*, let's launch an instance:
+
+      amc launch my-first-app --vm --enable-streaming
+
+This command creates and starts an instance for the application.
+
+`my-first-app` is the application name, you can also use the application ID to launch an instance for it.
+
+The `--enable-streaming` flag indicates that we want to stream this instance.
+
+The `--vm` flag indicates that we want to create the application inside a virtual machine instance instead of the default container instance.
+
+To see the instance details and its status, run:
+
+      amc ls
+
+When the instance status reaches the *running* status, let's try to access it. We will need to copy the instance ID from the previous command's output.
+
+      amc shell instance_id
+
+This takes us into the Linux instance that runs the Android container. Next, let's try accessing the nested Android container:
+
+      anbox-shell
+
+Now, we are inside the Android container. Play around by running some of the following commands:
+
+      ls
+      logcat
+      exit
+
+`exit` or pressing Control+D takes us back to the Linux container.
+
+**Update an application**
+
+We can have several versions of an application. Let's try and update the application:
+
+Go to the application directory and create a new manifest. This manifest introduces a change to the disk-size and also has a tag to associate with the application.
+
+```yaml
+name: my-first-app
+resources:
+   cpus: 4
+   memory: 4GB
+   disk-size: 8GB
+tags:
+   game
+```
+
+Now, run the following commands and observe the output:
+
+      amc application update my-first-app my-app/
+      amc application list
+      amc application show my-first-app
+
+The `list` command displays the list of applications available. As an application becomes *active*, it is automatically published and a new version of the `my-first-app` is created.
+
+The `show` command displays the application information along with its versions.
+
+Now if we launch our application without explicitly specifying a version, our latest published version will be considered.
+
+**Streaming the application**
+
+To look at a visual output of the stream-enabled instance that we created, we need a stream client.You can set up your own stream client by following the {ref}`tut-set-up-stream-client` later. For this tutorial, you can use the default client, the Anbox Cloud dashboard.
+
+Go to `https://machine-address` using a browser and go to the *Instances* page where we can see the instance we created. See the *Dashboard* tab for details on using the dashboard to test the virtual device.
```

Comment:
This may not work for the appliance, where the dashboard can only be accessed on `https://<machine-address>:5000`